### PR TITLE
Add missing "use function" declarations

### PR DESCRIPTION
**Pull request by sanmai**
This PR:

- [x] Adds missing "use function" declarations around the code
- [x] Adds missing "use constant" declarations around the code
- [x] Changes CS Fixer rules to fix these issues automatically.

### CODE REVIEW

File: bin/infection [hunk 1]
```diff
@@ -38,30 +38,30 @@ use Infection\Console\Application;
 use Infection\Container;
 use function Safe\getcwd;
 
-if (in_array(PHP_SAPI, ['cli', 'phpdbg', 'embed'], true) === false) {
-    echo PHP_EOL . 'Infection may only be invoked from a command line, got "' . PHP_SAPI . '"' . PHP_EOL;
+if (\in_array(\PHP_SAPI, ['cli', 'phpdbg', 'embed'], true) === false) {
```
Comment (theofidry) [inline bin/infection:41]: All of those could be use statements instead as well

File: .php_cs.dist [hunk 2]
```diff
@@ -36,11 +36,11 @@ declare(strict_types=1);
 use PhpCsFixer\Config;
 use PhpCsFixer\Finder;
 
-$header = trim(sprintf(
+$header = \trim(\sprintf(
```
Comment (theofidry) [inline .php_cs.dist:39]: same here

File: bin/infection-debug [hunk 3]
```diff
@@ -34,8 +34,8 @@
 
 declare(strict_types=1);
 
-set_error_handler(static function ($errno, $errstr): void {
-    var_dump($errno, $errstr);
-}, E_ALL);
+\set_error_handler(static function ($errno, $errstr): void {
```
Comment (theofidry) [inline bin/infection-debug:37]: same here

File: tests/xdebug-filter.php [hunk 4]
```diff
@@ -33,13 +33,13 @@
 
 declare(strict_types=1);
 
-if (!function_exists('xdebug_set_filter')) {
+if (!\function_exists('xdebug_set_filter')) {
     return;
```
Comment (theofidry) [inline tests/xdebug-filter.php:37]: same here

**Review by theofidry — APPROVED** [review]
LGTM otherwise

File: bin/infection (hunk 1 above)
Comment (sanmai) [reply bin/infection:41]: CS Fixer plainly refuses to make them so.

File: tests/xdebug-filter.php (hunk 4 above)
Comment (sanmai) [reply tests/xdebug-filter.php:37]: It seems like that's the cost of having this ruleset work in other places. It just won't make them imported here and there.

File: bin/infection (hunk 1 above)
Comment (maks-rafalko) [reply bin/infection:41]: I confirm - when I use import here and run `make cs`, it removes import and adds `\` 

```
1)) /home/maksrafalko/apps/infection/bin/infection (native_function_invocation, no_unused_imports)
```

On the other side, it works correctly for for example `SourceDirGuesser.php` - when I remove import there, fixer adds it back and doesn't add `\`:

```
1) /home/maksrafalko/apps/infection/src/Config/Guesser/SourceDirGuesser.php (native_function_invocation, global_namespace_import, ordered_imports)
```

it looks like a big in PHP-CS-Fixer

**Review by maks-rafalko — APPROVED** [review]

**Comment by maks-rafalko** [comment]
TravisCI is still broken. It shows we have 0 credits of 0 for the current month :| Still didn't get a reply from them, so I temporarily disabled TravisCI from protected branches so you can merge.

**Comment by sanmai** [comment]
I think it's a bout time to un-require Travis CI for protected branches.

**Comment by sanmai** [comment]
@maks-rafalko thank you!

**Comment by sanmai** [comment]
It seems like [this is the end of Travis  CI](https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works) we knew and loved.

**Comment by sidz** [comment]
RIP Travis.

**Pull request merged**